src/relay/backend/contrib/dnnl/codegen.cc

```diff
@@ -413,7 +413,8 @@ class DNNLModuleCodegen : public CSourceModuleCodegenBase {
     // Create a CSource module
     const auto* pf = runtime::Registry::Get("runtime.CSourceModuleCreate");
     ICHECK(pf != nullptr) << "Cannot find csource module to create the external runtime module";
-    return (*pf)(code, "c", sym, variables);
+    // TODO(@manupa-arm): pass the function names to enable system-lib creation
+    return (*pf)(code, "c", Array<String>{}, sym, variables);
```

Comment:
Do we need an array? it seems that a csourcemodule is only one function though.

Reply:
When we target "c", in it can have more than one function which also uses the same CSourceModuleCreate interface. See codegen_c_host.cc